packages/quantic/cypress/e2e/default-2/result-list/folded-result-list/folded-result-list.cypress.ts

```diff
@@ -139,11 +139,11 @@ describe('quantic-folded-result-list', () => {
     });
 
     describe('when the result has no child result', () => {
```

Comment:
I think it's more "when the result has no child collection" no?

Reply:
the child result is in the collection, but yeah I can change it to collection if you prefer. Can change that in my FTs PR though